### DISCUSSION TITLE
Do global scopes exist in v5?

### DISCUSSION THREAD

**Discussion by mcordoba** [adonisjs/core#1544] [Help]
In v4.1 we can add global scopes. Is this possible in v5 ?

I don't see an equivalente method in source code.

**Comment by thetutlage** [comment]
Nope not available. I also believe they are not required, since you can overwrite the `query` method and apply them yourself.

```ts
class User extends BaseModel {
  public static query() {
    return super.query()
  }
}
```

**Reply by blacktrue** [reply]
Hi @thetutlage 

Thinking soft deletions, I modify the query, but how would I ever consider the deleted ones?
